map/positions.txt

```diff
@@ -2493,35 +2493,35 @@
 #Sanctum of the Sun
 	357=
 	{
-		position={2880.000 1639.000 2872.000 1639.500 2875.000 1650.000 2876.000 1639.000 2881.000 1644.000 0.000 0.000 0.000 0.000}
+		position={2880.000 1639.000 2878.000 1636.000 2875.000 1650.000 2876.000 1639.000 2881.000 1644.000 0.000 0.000 0.000 0.000}
 		rotation={0.000 0.000 0.000 0.000 0.000 0.000 0.000}
 		height={0.000 0.000 0.000 20.000 0.000 0.000 0.000}
 	}
 #Farstrider Enclave
 	358=
 	{
-		position={2902.000 1656.000 2899.000 1649.000 2904.000 1664.000 2903.000 1659.000 2908.000 1664.000 0.000 0.000 0.000 0.000}
+		position={2902.000 1656.000 2900.000 1656.000 2904.000 1664.000 2903.000 1659.000 2908.000 1664.000 0.000 0.000 0.000 0.000}
 		rotation={0.000 0.000 0.000 0.000 0.000 0.000 0.000}
 		height={0.000 0.000 0.000 20.000 0.000 0.000 0.000}
 	}
 #Hatchet Hills
 	359=
 	{
-		position={2899.000 1628.000 2903.000 1632.000 2892.000 1622.000 2900.000 1623.000 2905.000 1628.000 0.000 0.000 0.000 0.000}
+		position={2899.000 1628.000 2900.000 1627.000 2892.000 1622.000 2900.000 1623.000 2905.000 1628.000 0.000 0.000 0.000 0.000}
 		rotation={0.000 0.000 0.000 0.000 0.000 0.000 0.000}
 		height={0.000 0.000 0.000 20.000 0.000 0.000 0.000}
 	}
 #Amani Pass
```

Comment:
Mb make it closer to lake?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/65841387-70ac9180-e32a-11e9-98a9-a09b46a403a5.png)

</details>

Reply:
Done!